packages/replay/src/integration.ts

```diff
@@ -226,6 +226,19 @@ Sentry.init({ replaysOnErrorSampleRate: ${errorSampleRate} })`,
     return this._replay.flushImmediate();
   }
 
+  /**
+   * If not in "session" recording mode, flush event buffer (i.e. creates a new replay).
+   * Unless `continueRecording` is false, the replay will continue to record and
+   * behave as a "session"-based replay.
+   */
+  public async capture(continueRecording: boolean = true): Promise<void> {
```

Comment:
So while I like this, I am not 100% sure about the API/naming/wording here.

My concern is that it may be a bit confusing/unclear to have `flush` and `capture` in their current forms.

For me, `capture` does not imply that this will only work for buffered sessions, and do nothing if recording is ongoing.
Also, `capture` maybe implies that it _starts_ capturing, maybe...? Not 100% sure.

Some alternative ideas:

* `captureBufferedSession` or something along these lines
* `sendBufferedSession` or something like this? As this is what is actually happening from a user perspective?

Maybe we could also rethink it a bit and make it e.g. `sendRecordingData` or something like this, where in the case of a `session` recording mode we just do `flushImmediately`, while else using the current behavior? It could still have an `continueRecording` flag (which is nice IMHO!). Maybe this would be a bit easier to reason about/document? WDYT?

Reply:
I agree -- what if we changed `flush` to be what you suggested IRT `sendRecordingData`?

Reply:
I think that makes sense! 👍 also one less additional API, and should not really be breaking as the main change is that it will continue recording when you've been in error mode I guess, which is probably rather expected anyhow. Great idea!